frontend/packages/console-shared/src/utils/resource-utils.ts

```diff
@@ -532,6 +549,30 @@ export class TransformResourceData {
     });
   };
 
+  getPipelineRunsForPipeline = (pipeline: Pipeline): PipelineRun[] => {
+    const PIPELINE_RUN_LABEL = 'tekton.dev/pipeline';
+    const pipelineRuns = _.get(this.resources, ['pipelineRuns', 'data']);
+    const pipelineName = pipeline.metadata.name;
+    return pipelineRuns.filter((pr: PipelineRun) => {
+      return (
+        pipelineName ===
+        (_.get(pr, ['metadata', 'labels', PIPELINE_RUN_LABEL], null) ||
+          _.get(pr, ['spec', 'pipelineRef', 'name'], null))
+      );
+    });
+  };
+
+  getPipelinesForResource = (resource: K8sResourceKind): PipelineOverviewItem => {
+    const pipelines = _.get(this.resources, ['pipelines', 'data']);
```

Comment:
Same concern here about `_.get`

Reply:
fixed in https://github.com/openshift/console/pull/3014/commits/3acef67b9193281dd53174625fd3d424051c286c